### ISSUE TITLE
Bug in stream content encoding of float numbers

### ISSUE THREAD

**Issue by surdus** [J-F-Liu/lopdf#116]
lopdf version: 0.25.0

Code to reproduce:
```rust
    let operations = vec![
        Operation { operator: "op".to_string(), operands: vec![Object::Real(0.0000001)] }
    ];

    let content = Content { operations };

    let encoded = content.encode()?;
    let decoded = Content::decode(&encoded)?;

    println!("{:?}", content);
    println!("{:?}", decoded);
    println!("{}", String::from_utf8(encoded)?);
```

Output:
```
Content { operations: [Operation { operator: "op", operands: [0.0000001] }] }
Content { operations: [Operation { operator: "e", operands: [1] }, Operation { operator: "op", operands: [-7] }] }
1e-7 op
```

Float number `0.0000001` encodes as `1e-7`. If you try to decode such number it decodes as new operation `1 e`. It also partially breaks PDF file.

**Comment by J-F-Liu** [comment]
Thanks for reporting, fixed by 2ba297a.

**Comment by Heinenen** [comment]
This issue seems to be fixed and can probably be closed.
